src/popup/components/form/FormTextarea.vue

```diff
@@ -45,17 +49,41 @@ export default {
     size: {
       type: String,
       default: null,
-      validator: (val) => SIZES.includes(val),
+      validator: (val: string) => SIZES.includes(val),
     },
+    autoHeight: Boolean,
   },
-  methods: {
-    handleEnterClick(event) {
-      if (!this.enterSubmit) return;
-      this.$emit('submit');
-      event.preventDefault();
-    },
+  setup(props, { emit }) {
+    const textarea = ref<HTMLTextAreaElement>();
+    const height = ref('auto');
```

Comment:
I see that you choose to set the `auto` or `px` values. I was mentioning on priv that this is the worse solution, as from my experience it's causing some issues (it's less predictable). Here's an example:
![image](https://user-images.githubusercontent.com/8217670/226908253-2285f4da-5fec-486b-864c-1ee8ef194a3a.png)
(the scrollbar is visible).